service/src/tests/mod.rs

```diff
@@ -19,24 +19,23 @@ use clap::ArgMatches;
 
 use crate::enclave::api::*;
 
-use self::{ecalls::*, integration_tests::*};
+use self::ecalls::*;
 use itp_enclave_api::enclave_test::EnclaveTest;
 
 pub mod commons;
 pub mod ecalls;
-pub mod integration_tests;
 pub mod mock;
 
 #[cfg(test)]
 pub mod worker;
 
 #[cfg(test)]
-pub mod enclave_api_mock;
+pub mod mocks;
```

Comment:
These `#[cfg(test)]` can also be removed, if the entire `tests` module has this attribute in `main.rs`.

Reply:
In main the test class ist used inside the code so we need to #[cfg(test)] this statement there. Is that ok?

Reply:
yes, so that is the one place where we need `#[cfg(test)]`: https://github.com/integritee-network/worker/blob/02c403e22c0f056042cf26143a9b9fcafbd32c8b/service/src/main.rs#L92 (at least as long as we only test the service with `cargo test`). If at one point we want to run the regular executable in an integration test mode (which we don't have at the moment), then we'll have to move those `#[cfg(test)]` around a bit.